invenio_app_ils/cli.py

```diff
@@ -661,6 +667,47 @@ def persist(self):
         return self.holder.related_records['objs']
 
 
+class DocumentRequestGenerator(Generator):
+    """Document requests generator."""
+
+    STATES = ["CANCELLED", "PENDING", "FULFILLED"]
```

Comment:
can you please move this to the `api` class? It is important that these states are defined there.

Reply:
Is this still relevant @equadon ?

Reply:
It's been handled in the frontsite PR.